source/Cosmos.System2/Graphics/SVGAIICanvas.cs

```diff
@@ -94,23 +95,33 @@ public override Mode Mode
         /// <exception cref="Exception">Thrown on memory access violation.</exception>
         public override void DrawPoint(Pen aPen, int aX, int aY)
         {
-            Color xColor = aPen.Color;
-
-            if (xColor.A == 0)
+            if (aPen.Color.A == 0)
             {
                 return;
             }
-            else if (xColor.A < 255)
+            else if (aPen.Color.A < 255)
             {
-                xColor = AlphaBlend(xColor, GetPointColor(aX, aY), xColor.A);
+                aPen.Color = AlphaBlend(aPen.Color, GetPointColor(aX, aY), aPen.Color.A);
             }
 
-            mSVGAIIDebugger.SendInternal($"Drawing point to x:{aX}, y:{aY} with {xColor.Name} Color");
-            _xSVGADriver.SetPixel((uint)aX, (uint)aY, (uint)xColor.ToArgb());
-            mSVGAIIDebugger.SendInternal($"Done drawing point");
-            /* No need to refresh all the screen to make the point appear on Screen! */
-            //xSVGAIIDriver.Update((uint)x, (uint)y, (uint)mode.Columns, (uint)mode.Rows);
-            _xSVGADriver.Update((uint)aX, (uint)aY, 1, 1);
+            _xSVGADriver.SetPixel((uint)aX, (uint)aY, (uint)aPen.ValueARGB);
+        }
+
+        public void DrawPointFast(uint aColor, uint aX, uint aY)
```

Comment:
Why is this called `DrawPointFast` and not just `DrawPoint` with a different set of parameters?